frontend/src/components/fields/fieldConfirmation/FieldConfirmation.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {FieldConfirmationProps} from "./props";
+import {InputAdornment, TextField} from "@mui/material";
+import {DISABLE_FIELD} from "../../../common/constansts/fields";
+
+const FieldConfirmation: React.FC<FieldConfirmationProps> = ({data, prefix}) => {
```

Comment:
bad name. This component has nothing about confirmation. This is just text field with is disabled all the time has some prefix for value